index.html

```diff
@@ -37,13 +40,150 @@ <h1 class="supporting-text">I'm a software developer</h1>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
 
+    </section>
 
-    </ul>
+  </section>
+  <section class="about-me-section">
+    <div class="about-me">
+      <h3 class="about-title">About me</h3>
+      <p class="description-about">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </div>
+    <hr class="solidhr1">
+    <div class="skills">
+      <div class="about-me-sectiones">
+        <div class="holder-image">
+          <img src="images/triangle.png" alt="">
+          <h3 class=" format">Language</h3>
+        </div>
+        <ul class="about-languages">
+          <li class="des-text">Javascript</li>
+          <li class="des-text">HTML</li>
+          <li class="des-text">CSS</li>
+        </ul>
+      </div>
+      <div class="about-me-sectiones">
+        <div class="holder-image">
+          <img src="images/rectangle.png" alt="">
+          <h3 class="format">Frameworks</h3>
+        </div>
```

Comment:
- [ ] Its awesome that you make technology in each category(e.g., "JavasScript", "HTML" and "CSS") a `ul `with each being inside `li`, But we also need to follow the same structure for technology categories ("Languages", "Frameworks" and "Skills"). Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`